javascript/code-challenges/challenges-03.test.js

```diff
@@ -35,9 +35,8 @@ Write a function named containsAnd that, given an array of strings as input, use
 
 For example, containsAnd(['panda', 'ran', 'and']) returns ['panda', 'and'].
 ------------------------------------------------------------------------------------------------ */
-
 const containsAnd = (arr) => {
-
+  return arr.filter(Strings => Strings.includes('and'));
```

Comment:
All of the challenges can be solved like this, directly returning arr.filter rather than putting it inside a variable first.